src/js/common/renderer.js

```diff
@@ -121,7 +121,8 @@ var Handlebars = Handlebars || {};
         defaultLocale: "en_us",
         defaultLayout: "main",
         events: {
-            onTemplatesLoaded: null
+            onTemplatesLoaded: null,
```

Comment:
We should discuss whether there should be an `onMessagesLoaded` event here as well, however we agree to handle things, the approach should be reflected in the common grade as well.